tests/e2e/cypress/fixtures/eligibility.json

```diff
@@ -0,0 +1,14 @@
+{
+  "eligible": {
+    "name": "Garcia",
+    "sub": "A1234567"
+  },
+  "wrongName": {
```

Comment:
Maybe call this one `ineligible` to pair with `eligible`. Since it's not technically "wrong" like the one below is.